src/translations/translations_EN.json

```diff
@@ -522,6 +522,7 @@
     "reporting": {
       "report_resource": "Report resource",
       "report_sent_feedback": "Report sent",
+      "doi_request_sent_feedback": "Request for DOI has been sent",
```

Comment:
Vi forsøker å ha oversettelsene alfabetisk. Så linje 525 skal flyttes til 523. (samme for engelsk)

Reply:
so it shall be